arangod/Cluster/ClusterComm.h

```diff
@@ -688,7 +689,7 @@ class ClusterCommThread : public Thread {
  public:
   ClusterCommThread();
   ~ClusterCommThread();
-
+  void halt();
```

Comment:
we can probably remove this again, and instead call the (adjusted) shutdown method.